Classes/Plugins/Ads/PKAdInfo.swift

```diff
@@ -46,21 +47,6 @@ import GoogleInteractiveMediaAds
         }
     }
     
-    init(ad: IMAAd) {
```

Comment:
@gal-orlanczyk  let's make sure the docs are updated.
And what about the issue you resolved yesterday based on information event?